apps/root/src/pages/home/components/activity/index.tsx

```diff
@@ -155,15 +157,17 @@ const ActivityContent: ItemContent<TransactionEvent, Context> = (
         <StyledBodySmallLabelTypography noWrap={false}>{formattedDate}</StyledBodySmallLabelTypography>
       </StyledOperation>
       <StyledValue>
-        <StyledBodySmallRegularTypo2 noWrap={false}>{txTokenFlow}</StyledBodySmallRegularTypo2>
+        <StyledBodySmallRegularTypo2 noWrap={false}>
+          {showBalances ? txTokenFlow : <HiddenNumber size="small" />}
+        </StyledBodySmallRegularTypo2>
```

Comment:
The UI changes a little when toggling showBalances, I think we can solve it by changing <HiddenNumber to be outside the <Typography /> and increasing its line height a little to be 18px

https://github.com/Balmy-protocol/dca-fe/assets/144699868/bee26fba-81f3-4c13-a265-3b141ab1d5de

Reply:
I mean it's going to change based on that now the bottom date has more space to expand, I don't think it is bad